torch/onnx/symbolic_opset9.py

```diff
@@ -1041,6 +1041,21 @@ def ne(g, self, other):
     return g.op("Equal", self, other)
 
 
+def __isnot_(g, self, other):
+    if sym_help._is_none(other):
+        if sym_help._is_none(self):
+            return g.op("Constant", value_t=torch.BoolTensor([1]))
+        return g.op("Constant", value_t=torch.BoolTensor([0]))
+    return ne(g, self, other)
```

Comment:
Will end up at this line if `other` is not `None`, and self is `None`. Is this possible?